txtl-liposome_water-in-oil.md

```diff
@@ -0,0 +1,217 @@
+
+#### Materials:
+
+* 20 uM of HPTS stock solution: ([https://www.thermofisher.com/order/catalog/product/H348](https://www.thermofisher.com/order/catalog/product/H348))
+
+* 100 mM HEPES, 200 mM glucose pH 8
+
+* 2M Sucrose Stock
+
+* 100 mM HEPES + 250 mM glucose, pH 8
+
+* 100 mM HEPES, 200 mM sucrose pH 8
+
+### **Step 3**: Microscopy Visualization
+
+#### Materials:
+
+* Frame Seals: [https://www.thermofisher.com/order/catalog/product/S24736](https://www.thermofisher.com/order/catalog/product/S24736)
+
+* Glass slides
+
+## Step 0: Deposition of a thin lipid film in a glass vials
+
+The first step in the protocol is to generate the lipid film required
+to to form the lipid-in-mineral oil solution. Makes 6 samples with
+accurate measurement. Each vial will have ~ 15 mg of lipid with 0.1
+mol % of 18:1 Lyss-Rho-PE.
+
+1. Create lipid master mix in a glass beaker 
+
+    a. POPC/Lyss-Rhod-PE 
+
+        i. Add  4 mL (100 mg) of POPC in chloroform (25 mg/ml) 
+
+        ii. Add 200 uL (0.2 mg) of Lyss-Rho-PE in chloroform (1 mg/mL) 
+
+        iii. Swirl gently until all POPC is dissolved and color is homogeneous.
+
+    b. POPC/Cholesterol/Lyss-Rhod-PE 
+
+        i. Add 2.668 mL (66.7 mg) of POPC in chloroform (25 mg/ml) 
+
+        ii. Add 1.332 mL (33.3 mg) of Cholesterol in chloroform (25 mg/ml) 
+
+        iii. Add 200 uL (0.2 mg) of Lyss-Rho-PE in chloroform (1 mg/mL) 
+
+        iv. Swirl gently until all POPC is dissolved and color is homogeneous.
+
+2. Aliquot 700 uL of POPC/Cholesterol/Lyss-Rhod-PE Chloroform solution
+into 2 mL glass vials (6 vials total)
+
+3. Place uncapped vials in fume hood, loosely covered with aluminum
+foil and allow to evacuate overnight in fume hood (~6-8 hours).
+
+    * **Note:** The purpose of the aluminum foil cover is to protect
+      particles from contaminating the POPC/Cholesterol/Lyss-Rhod-PE
+      as it evaporates.
+
+4. Move vials to vacuum chamber and vacuum for an additional 2 hours..
+
+5. Store in -20 degC.
+
+**Note:** Remaining POPC in chloroform, cholesterol, and  Lyss-Rho-PE in chloroform can be stored at -20 degC in the glass vial with PTFE caps and sealed parafilm.
+
+## Step 1: Preparation of the oil containing lipids
+
+1. Mix mineral oil by gentle inversion before use 
+
+2. Place 0.5 mL of mineral oil into each of the vials.
+
+3. Incubate at 60 degC  for 10 min
+
+4. Vortex for 10 mins
+
+5. Incubate vials for 3 hrs at 60 degC.
+
+6. Wrap up the top of the vials with aluminum foil and seal with parafilm.
+
+7. Sonicate in an heated water bath for 30 minutes,at 60 degC .
+
+    a. **Note:** If a sonicator is unavailable then repeat 60 degC
+    incubation and vortexing until the lipid is completely dissolved.
+
+## Step 2: Self-assembly of the Liposomes by centrifugation
+
+1. Place 225 μL of centrifuge buffer (100 mM HEPES + 200 mM glucose,
+   pH 8) into a labeled eppendorf tube.
+
+    a. **Note**: When IVTL systems are used, the outer solution should
```

Comment:
IVTL is not defined.  I assume this means "in vitro translation system"?  See issue #23 for some discussion.